### PR TITLE
Static components for PDF

### PR DESCRIPTION
**Pull request by anncatton**
WIP for PDF feature. On recommendation from @blabadi , getting this portion into dev sooner rather than later to kickstart feedback
- adds components for static text portions of `Introduction` and `Applicant` sections
- adds some simple styled components for pdf display
- generates pdf on demand. api call for app data made in this onClick function so data is most recently saved state
- setup for form text still *very* wip, as i haven't completely decided on the best approach for this (so feedback is very welcome!). PDF text for the form inputs is not always the same as the UI
- have added only a few data bits to the pdf to test it is working (applicant info, introduction acknowledgement)
- will be adding type info as work progresses
- addresses portions of #127 , #128 and #60

### CODE REVIEW

File: components/Head.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ const Head = () => {
   return (
     <NextHead>
       <link
-        href={'https://fonts.googleapis.com/css?family=Lato:300,400,600&display=swap'}
+        href={'https://fonts.googleapis.com/css?family=Work+Sans:300,400,600&display=swap'}
```
Comment (samrichca) [inline components/Head.tsx:8]: just checked, daco has this too. i'll correct it on my next PR :P

Comment (anncatton) [reply components/Head.tsx:8]: yeah i forgot to note that in the pr description...i believe this was my mistake! head still in overture space 🙂

File: components/pages/Applications/ApplicationForm/Header/Actions.tsx [hunk 2]
```diff
@@ -21,8 +42,21 @@ const HeaderActions = (): ReactElement => {
...
-
-      <Button onClick={function noRefCheck() {}} size="sm" variant="secondary">
+      <Button
+        isAsync
+        variant="secondary"
+        size="sm"
+        onClick={async () => {
+          const data = await fetchWithAuth({ url: urlJoin(APPLICATIONS_PATH, appId) })
+            .then((res: any) => {
+              if (res.status === 200) {
```
Comment (samrichca) [inline components/pages/Applications/ApplicationForm/Header/Actions.tsx:52]: I'm gonna add validateStatus 200 to fetchWithAuth, so the status check can be removed here

File: components/pages/Applications/ApplicationForm/Header/Actions.tsx [hunk 3]
```diff
@@ -21,8 +42,21 @@ const HeaderActions = (): ReactElement => {
...
+                return res.data;
+              }
+            })
+            .catch((err: any) => console.log('ERROR! ', err));
```
Comment (samrichca) [inline components/pages/Applications/ApplicationForm/Header/Actions.tsx:56]: console.warn instead?

Comment (anncatton) [reply components/pages/Applications/ApplicationForm/Header/Actions.tsx:56]: yes ma'am! forgot about this one, nice catch :)

**Review by samrichca — APPROVED** [review]
this pdf solution looks really good!! nice work

File: components/pages/Applications/ApplicationForm/Header/Actions.tsx (hunk 3 above)
Comment (samrichca) [reply components/pages/Applications/ApplicationForm/Header/Actions.tsx:56]: catching it again........ we're using console.error in other fetch errors... my b

**Review by blabadi — APPROVED** [review]

**Review by caravinci — APPROVED** [review]
Looks great to me 🚀

**Pull request merged**